core/src/main/scala/org/apache/spark/scheduler/cluster/ExecutorData.scala

```diff
@@ -27,12 +27,14 @@ import org.apache.spark.rpc.{RpcAddress, RpcEndpointRef}
  * @param executorHost The hostname that this executor is running on
  * @param freeCores  The current number of cores available for work on the executor
  * @param totalCores The total number of cores available to the executor
+ * @param pcis The external devices available to the executor
```

Comment:
I would name it a bit more generic than `pcis` since PCI is just an interface name that could change later.